kinitro/api/routes/tasks.py

```
@@ -0,0 +1,138 @@
+                success=result.success,
+                score=result.score,
+                total_reward=result.total_reward,
+                timesteps=result.timesteps,
+                error=result.error,
+            )
+            if success:
+                accepted += 1
+            else:
+                rejected += 1
+                errors.append(f"Task {result.task_id}: not found or not assigned to executor")
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)